chef/cookbooks/hyperv/attributes/default.rb

```diff
@@ -6,6 +6,9 @@
   "Microsoft-Hyper-V" => {
     "restart" => false
   },
+  "Hyper-V-Tools" => {
```

Comment:
Apparently, this needs to be changed to RSAT-Hyper-V-Tools-Feature. @ovisan can you confirm?